sheetstorm/settings/base.py

```diff
@@ -268,3 +268,5 @@
 ]
 
 CRISPY_TEMPLATE_PACK = 'bootstrap4'
+
+VALID_EMAIL_DOMAIN_LIST = ["codepoets.it"]
```

Comment:
Also for this setting should be added `system_check` that this setting is correctly set. [Here](https://github.com/golemfactory/concent/blob/master/concent_api/concent_api/system_check.py) you can see how `system_check`s should be created. I think, that we should also add system checks for other settings. I can see that there are two settings which can be checked - `SITE_ID` and `EMAIL_BACKEND` which are used by `production.py` settings. What do you think guys about this @rwrzesien @dybi ?

Reply:
Sounds like a good exercise, lets do it, but maybe in separate issue (but assigned to @Szymiks too, so he can learn this)?

Reply:
So after this PR I will make this issue

Reply:
Separate issue sounds good :)